### PR TITLE
Multiplayer status

### PR DESCRIPTION
**Pull request by eniklas**
Previously, we were using `max_players` to decide whether a game is multiplayer or not, which is problematic when we don't know that value. With this change, we now set a boolean `multiplayer` for each title, which is `True` if and only if:
- max players > 1 (no matter how that number was reached)
- the game modes in IGDB include a multiplayer mode

We won't filter games that we have no multiplayer info on, which I think is preferable because we're better off assuming a game is multiplayer if we know nothing about it; it's easier to catch them as the multiplayer list is much shorter, and less impactful to see a few extra games we don't care about rather than omitting games we do care about.

I also got rid of the `include_zero_players` option, as it was a complication only needed because of the previous method.

### CODE REVIEW

File: gamatrix-gog.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+# TODO: copy max players to other games with the same sanitized title
```
Comment (derek-keeler) [inline gamatrix-gog.py:2]: Don't bother leaving this, just make an issue!

File: gamatrix-gog.py [hunk 2]
```diff
@@ -194,21 +198,27 @@ def build_config(args):
...
 
         if "max_players" in game_list[k]:
-            log.debug(
-                f'{k}: max players {game_list[k]["max_players"]} from config file'
-            )
-            continue
+            max_players = game_list[k]["max_players"]
+            reason = "from config file"
+            if max_players > 1:
+                multiplayer = True
```
Comment (derek-keeler) [inline gamatrix-gog.py:221]:
```suggestion
            multiplayer = max_players > 1
```

**Review by derek-keeler — APPROVED** [review]
LGTM

File: gamatrix-gog.py (hunk 1 above)
Comment (eniklas) [reply gamatrix-gog.py:2]: Yeah, was being lazy 😞 About to do that next, will remove on the next PR.

File: gamatrix-gog.py (hunk 2 above)
Comment (eniklas) [reply gamatrix-gog.py:221]: Oh yeah, nice one

**Pull request merged**